vmslave-config/ansible/group_vars/all

```diff
@@ -0,0 +1,14 @@
+# If not set, default: use roles/replace-apt-source/files/sources.list
+sources_list_url: http://**.**.**.**/static/misc/sources.list
+
+# vagrant box packages
+vagrant_deb: https://releases.hashicorp.com/vagrant/1.9.1/vagrant_1.9.1_x86_64.deb
+virtualbox_deb: http://download.virtualbox.org/virtualbox/4.3.36/virtualbox-4.3_4.3.36-105129~Ubuntu~raring_amd64.deb
+
+# ovftool bundle url
+ovftool_bundle: http://ftp.tucha13.net/pub/software/VMware-ovftool-4.1.0/VMware-ovftool-4.1.0-2459827-lin.x86_64.bundle
+# vars for mount isofarm
+isofarm_path: /home/jenkins/isofarm
```

Comment:
isofarm is "rackhd ci private" and strong binding to our pr gate.
I think community  doesn't  need to understand "ci system strong binding" things.
What's your advice?